sql/migrations/005_resolution_creation_timestamp.sql

```diff
@@ -0,0 +1,10 @@
+-- +migrate Up
+
+-- Adds the resolution creation timestamp on resolution table and sets its value to last_start
+ALTER TABLE "resolution" ADD COLUMN "created" TIMESTAMP with time zone;
+UPDATE "resolution" SET created = last_start;
+ALTER TABLE "resolution" ALTER COLUMN "created" SET NOT NULL;
```

Comment:
I had a new thought about the migration procedure: if we don't have the `DEFAULT now();` here, then once the SQL migration is run, all the µTask must run on the new version, otherwise any `INSERT` into the `resolution` table will fail, due to the `NOT NULL` constraint.

Also, that's the implementation inside the `sql/schema.sql`.

```suggestion
ALTER TABLE "resolution" ALTER COLUMN "created" SET NOT NULL, ALTER COLUMN "created" SET DEFAULT now();
```